conda/base/constants.py

```diff
@@ -266,6 +266,12 @@ class SatSolverChoice(ValueEnum):
 DEFAULT_SOLVER = "libmamba"
 CLASSIC_SOLVER = "classic"
 
+#: The name of the default json reporter backend
+DEFAULT_JSON_REPORTER_BACKEND = "json"
+
+#: The name of the default console reporter backend
+DEFAULT_CONSOLE_REPORTER_BACKEND = "classic"
```

Comment:
I don't think "classic" fits here as a name since that would imply it would not be offered anymore like the classic solver, which is a concrete class etc, while the default console rendering is just built-in.

How about this?

```suggestion
DEFAULT_CONSOLE_REPORTER_BACKEND = "default"
```